schema/bom-1.4-SNAPSHOT.proto

```diff
@@ -450,3 +452,179 @@ message Evidence {
+  IMPACT_ANALYSIS_STATE_FALSE_POSITIVE = 4;
+  // The component or service is not affected by the vulnerability. Justification should be specified for all not_affected cases.
+  IMPACT_ANALYSIS_STATE_NOT_AFFECTED = 5;
+}
+
+enum ImpactAnalysisJustification {
+  // The code has been removed or tree-shaked.
+  IMPACT_ANALYSIS_JUSTIFICATION_CODE_NOT_PRESENT = 0;
+  // The vulnerable code is not invoked at runtime.
+  IMPACT_ANALYSIS_JUSTIFICATION_CODE_NOT_REACHABLE = 1;
+  // Exploitability requires a configurable option to be set/unset.
+  IMPACT_ANALYSIS_JUSTIFICATION_REQUIRES_CONFIGURATION = 2;
+  // Exploitability requires a dependency that is not present.
+  IMPACT_ANALYSIS_JUSTIFICATION_REQUIRES_DEPENDENCY = 3;
+  // Exploitability requires a certain environment which is not present.
+  IMPACT_ANALYSIS_JUSTIFICATION_REQUIRES_ENVIRONMENT = 4;
+  // Exploitability requires a compiler flag to be set/unset.
+  IMPACT_ANALYSIS_JUSTIFICATION_PROTECTED_BY_COMPILER = 5;
+  // Exploits are prevented at runtime.
+  IMPACT_ANALYSIS_JUSTIFICATION_PROTECTED_AT_RUNTIME = 6;
+  // Attacks are blocked at physical, logical, or network perimeter.
+  IMPACT_ANALYSIS_JUSTIFICATION_PROTECTED_AT_PERIMETER = 7;
+  // Preventative measures have been implemented that reduce the likelihood and/or impact of the vulnerability.
+  IMPACT_ANALYSIS_JUSTIFICATION_PROTECTED_BY_MITIGATING_CONTROL = 8;
+}
+
+enum VulnerabilityResponse {
+  VULNERABILITY_RESPONSE_CAN_NOT_FIX = 0;
+  VULNERABILITY_RESPONSE_WILL_NOT_FIX = 1;
+  VULNERABILITY_RESPONSE_UPDATE = 2;
+  VULNERABILITY_RESPONSE_ROLLBACK = 3;
+  VULNERABILITY_RESPONSE_WORKAROUND_AVAILABLE = 4;
+}
+
+message VulnerabilityAffects {
+  // References a component or service by the objects bom-ref
+  string ref = 1;
+  // Zero or more individual versions or range of versions.
+  repeated VulnerabilityAffectedVersions versions = 2;
+}
+
+message VulnerabilityAffectedVersions {
+  oneof choice {
+    // A single version of a component or service.
+    string version = 1;
+    // A version range specified in Package URL Version Range syntax (vers) which is defined at https://github.com/package-url/purl-spec/VERSION-RANGE-SPEC.rst
+    string range = 2;
+  }
+  // The vulnerability status for the version or range of versions.
+  optional VulnerabilityAffectedStatus status = 3;
+}
+
+enum VulnerabilityAffectedStatus {
+  // The vulnerability status of a given version or range of versions of a product. The statuses 'affected' and 'unaffected' indicate that the version is affected or unaffected by the vulnerability. The status 'unknown' indicates that it is unknown or unspecified whether the given version is affected. There can be many reasons for an 'unknown' status, including that an investigation has not been undertaken or that a vendor has not disclosed the status.
+  VULNERABILITY_AFFECTED_STATUS_AFFECTED = 0;
+  VULNERABILITY_AFFECTED_STATUS_NOT_AFFECTED = 1;
+  VULNERABILITY_AFFECTED_STATUS_UNKNOWN = 2;
+}
```

Comment:
Shift UNKNOWN to default 0 value

Reply:
Changed in 0e1df61cef885ce82273b5afa0295358e1ca9c5a